.github/workflows/algolia-integrity.yml

```diff
@@ -0,0 +1,56 @@
+# Note: this is a shared pipeline used by other repositories. 
+# Docs: https://docs.github.com/en/actions/using-workflows/reusing-workflows
+
+on:
+  workflow_call:
+    secrets:
+      domain:
+        required: true
+      lockFilePath:
+        required: true
+      docsBaseDir:
+        required: true
+
+jobs:
+  algolia-records-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Fetch
+        run: git fetch origin master
+
+      - name: Setup env
+        uses: the-guild-org/shared-config/setup@main
+        with:
+          nodeVersion: 18
+
+      - name: Algolia generate
+        run: yarn workspace website theguild-nextra-algolia -o ${{ secrets.lockFilePath }} --domain ${{ secrets.domain }} --docsBaseDir ${{ secrets.docsBaseDir }}
+
+      - name: Prettier
+        run: yarn prettier
+
+      - name: Compare
```

Comment:
Do we really need to run `diff` twice? 
I guess the goal here was to output it to both console AND a file? If so, let's output it to a file, and then do `cat FILE` in another step, instead of running it twice?